packages/block-library/src/post-author/edit.js

```diff
@@ -205,12 +202,6 @@ function PostAuthorEdit( {
 									className="wp-block-post-author__byline"
 									multiline={ false }
 									placeholder={ __( 'Write byline …' ) }
-									withoutInteractiveFormatting
-									allowedFormats={ [
-										'core/bold',
-										'core/italic',
-										'core/strikethrough',
-									] }
```

Comment:
What's the reasoning behind this change?

Reply:
I figured that if the goal was to have parity with paragraph block's functionalities, that the editable text should be able to have links and the remaining text formatting options.  Removing these removes the restrictions and enables them.

Reply:
I think these specific restrictions were put in place because other formatting options don't make sense in the context of a byline.

Reply:
Why restrict it thought?  If a user wants a word in their byline to be a link, then all of a sudden a link makes sense in the context of a byline for them.  A similar argument for subscript/superscript, etc.